src/Banner.js

```diff
@@ -62,6 +62,7 @@ const Banner = props => {
         <Box width={1} align={props.textAlign}>
           <Heading.h5>{props.header}</Heading.h5>
           <Text.span fontSize={1}>{props.text}</Text.span>
+          {props.children}
```

Comment:
Would the implementation from my PR suit your needs?

https://github.com/pricelinelabs/design-system/pull/145/files#diff-fa3de661379cadd7a682d81cecc3699aR64

Reply:
Oh cool didn't see that. I think it's more intuitive to use props.children but I guess that's not up to me.